man/systemd-nspawn.xml

```diff
@@ -527,14 +527,14 @@
       </varlistentry>
 
       <varlistentry>
-        <term><option>-E <replaceable>NAME</replaceable>=<replaceable>VALUE</replaceable></option></term>
-        <term><option>--setenv=<replaceable>NAME</replaceable>=<replaceable>VALUE</replaceable></option></term>
-
-        <listitem><para>Specifies an environment variable assignment
-        to pass to the init process in the container, in the format
-        <literal>NAME=VALUE</literal>. This may be used to override
-        the default variables or to set additional variables. This
-        parameter may be used more than once.</para></listitem>
+        <term><option>-E <replaceable>NAME</replaceable>[=<replaceable>VALUE</replaceable>]</option></term>
+        <term><option>--setenv=<replaceable>NAME</replaceable>[=<replaceable>VALUE</replaceable>]</option></term>
+
+        <listitem><para>Specifies an environment variable assignment to pass to the init process in the
+        container, in the format <literal>NAME=VALUE</literal>. This may be used to override the default
+        variables or to set additional variables. This parameter may be used more than once. When
+        <replaceable>VALUE</replaceable> is not specified, the current value of the variable with the
```

Comment:
this is misleading. Omitting VALUE is not sufficient to trigger the behaviour. You must omit `=` *and* VALUE